python/sglang/srt/managers/io_struct.py

```diff
@@ -397,6 +397,8 @@ def __getitem__(self, i):
                 else None
             ),
             return_hidden_states=self.return_hidden_states,
+            bootstrap_host=self.bootstrap_host[i],
```

Comment:
if not a list, then self.bootstrap_host

Reply:
It seems we will always make it a list in mini_lb when batch request, thus wondering whether we should do this here

Reply:
you are right. i misread.